get_books.py

```diff
@@ -158,6 +166,7 @@ def scrape_book(book_id):
 
     return {'book_id_title':        book_id,
             'book_id':              get_id(book_id),
+            'cover_image':          get_cover_image(soup),
```

Comment:
Recommend this be renamed, append `_uri` for consistency (see line 172) and clarity.